### ISSUE TITLE
Second `/shutdown` blackscreens on the FFXI -> POL transition

### ISSUE THREAD

**Issue by Byrth** [Windower/Fenestra#8]
If I am logged in with two accounts on windower 5 (no addons) and use `/shutdown` on both of them, the first one successfully transitions from FFXI -> POL -> desktop.  However, the second one transitions from FFXI -> POL and hangs.

I have observed this four times at this point. It is eminently repeatable.

The one time that my first instance crashed to desktop (missing the crash handler), the second instance was able to `/shutdown` without hanging.

**Comment by Byrth** [comment]
Other details:
1) It is login order independent
2) The blackscreen PoL window can be resized, has the chocobo cursor, etc. so it isn't frozen but it's just not loading.

**Comment by maka85** [comment]
i use windower 4 and it does the same thing hello yall im just checking yalls github im interested in programming again i used to do it for fun as a kid but it was like :"What to make" now on FFXI the possibilities are endless. i don't know how any of the new windows work last one i used was 2000xp but i  knew a lil C++, Delphi, Visual Basic , Basic A, Q basic , and turbo paschal . right now i don't even know how to get superwarp to load but i made my own outta scripts im eager to learn and i love the challenge. i can be a assistant doing what yall tell me to do

**Comment by Nifim** [comment]
@maka85 the issue tracker is not the best place for these conversations. if you are interested in helping you can join our [discord](https://discord.com/channels/338590234235371531/633140308850376714), there we discuss a sorts of stuff related to our development and help people with their own projects as well.

**Comment by maka85** [comment]
I would love the chance to join I am actually quite honored and thank you
for responding so fast I'm on the window or discord already but I don't
know if this is a new discord that y'all have for developers and thank you
again for reaching out to me I appreciate it                -Maka

**Comment by z16** [comment]
Nah, we use the same Discord server. We have some internal channels, reserved for the development team, and there's a public development channel, but access is guarded by roles - see the `#welcome` channel for more information. You can just contact us there.

**Comment by maka85** [comment]
Mine still does the exact thing everytime I shutdown. As long as there is only one official shutdown it seems the other will hang. Maybe something with how the profile saves on exit? Logging to one file blocking another.? I don’t understand computers only a lil bit of programming ty for CCing me. Sent from Mail for Windows From: ByrthSent: Friday, December 23, 2022 8:29 AMTo: Windower/FenestraCc: maka85; MentionSubject: [Windower/Fenestra] Second `/shutdown` blackscreens on the FFXI -> POL transition (#8) If I am logged in with two accounts on windower 5 (no addons) and use /shutdown on both of them, the first one successfully transitions from FFXI -> POL -> desktop. However, the second one transitions from FFXI -> POL and hangs.I have observed this four times at this point. It is eminently repeatable.The one time that my first instance crashed to desktop (missing the crash handler), the second instance was able to /shutdown without hanging.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
